<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1250</width>
    <height>851</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>主界面</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>170</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>请确认场景：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="audioSettingButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>480</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>  声音设置</string>
    </property>
   </widget>
   <widget class="QPushButton" name="advancedSettingButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>630</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>  高级功能</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cdnSettingButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>680</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>  混流设置</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>-100</y>
      <width>31</width>
      <height>2000</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QCheckBox" name="strRoomIdCheckBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>130</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Big Caslon</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>支持字符串类型</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="sdkVersionLabel_11">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>60</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>请输入用户ID：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="userNameLineEdit">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>60</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>请输入用户ID</string>
    </property>
   </widget>
   <widget class="QPushButton" name="enterRoomButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>310</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>进房</string>
    </property>
   </widget>
   <widget class="QPushButton" name="generalSettingButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>430</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>  常规设置</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>210</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>请确认角色：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="roleComboBox">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>210</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>主播</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>观众</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="screenShareSettingButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>580</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>  屏幕分享</string>
    </property>
   </widget>
   <widget class="QLabel" name="sdkVersionLabel_12">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>100</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>请输入房间号：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="appSceneComboBox">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>170</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>视频通话场景</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>视频互动直播</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>语音通话场景</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>语音互动直播</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="leaveRoomButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>310</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>退房</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="roomNameLineEdit">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>100</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>请输入房间号</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="videoSettingButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>530</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>  视频设置</string>
    </property>
   </widget>
   <widget class="QListView" name="listView">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>291</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QListView" name="listView_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>290</y>
      <width>291</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QListView" name="listView_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>410</y>
      <width>291</width>
      <height>381</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>30</y>
      <width>831</width>
      <height>751</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="bgmSettingButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>730</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>  BGM设置</string>
    </property>
   </widget>
   <zorder>listView_3</zorder>
   <zorder>listView_2</zorder>
   <zorder>listView</zorder>
   <zorder>label_5</zorder>
   <zorder>audioSettingButton</zorder>
   <zorder>advancedSettingButton</zorder>
   <zorder>cdnSettingButton</zorder>
   <zorder>line</zorder>
   <zorder>strRoomIdCheckBox</zorder>
   <zorder>sdkVersionLabel_11</zorder>
   <zorder>userNameLineEdit</zorder>
   <zorder>enterRoomButton</zorder>
   <zorder>generalSettingButton</zorder>
   <zorder>label_6</zorder>
   <zorder>roleComboBox</zorder>
   <zorder>screenShareSettingButton</zorder>
   <zorder>sdkVersionLabel_12</zorder>
   <zorder>appSceneComboBox</zorder>
   <zorder>leaveRoomButton</zorder>
   <zorder>roomNameLineEdit</zorder>
   <zorder>videoSettingButton</zorder>
   <zorder>widget</zorder>
   <zorder>bgmSettingButton</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1250</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
